src/map/entities/mobentity.h

```diff
@@ -51,18 +51,18 @@ enum SPECIALFLAG
 
 enum ROAMFLAG : uint16
 {
-    ROAMFLAG_NONE    = 0x00,
-    ROAMFLAG_NONE0   = 0x01,  //
-    ROAMFLAG_NONE1   = 0x02,  //
-    ROAMFLAG_NONE2   = 0x04,  //
-    ROAMFLAG_NONE3   = 0x08,  //
-    ROAMFLAG_NONE4   = 0x10,  //
-    ROAMFLAG_NONE5   = 0x20,  //
-    ROAMFLAG_WORM    = 0x40,  // pop up and down when moving
-    ROAMFLAG_AMBUSH  = 0x80,  // stays hidden until someone comes close (antlion)
-    ROAMFLAG_EVENT   = 0x100, // calls lua method for roaming logic
-    ROAMFLAG_IGNORE  = 0x200, // ignore all hate, except linking hate
-    ROAMFLAG_STEALTH = 0x400  // stays name hidden and untargetable until someone comes close (chigoe)
+    ROAMFLAG_NONE     = 0x00,
+    ROAMFLAG_NONE0    = 0x01,  //
+    ROAMFLAG_NONE1    = 0x02,  //
+    ROAMFLAG_NONE2    = 0x04,  //
+    ROAMFLAG_NONE3    = 0x08,  //
+    ROAMFLAG_NONE4    = 0x10,  //
+    ROAMFLAG_NONE5    = 0x20,  //
+    ROAMFLAG_WORM     = 0x40,  // pop up and down when moving
+    ROAMFLAG_AMBUSH   = 0x80,  // stays hidden until someone comes close (antlion)
+    ROAMFLAG_SCRIPTED = 0x100, // calls lua method for roaming logic
```

Comment:
We need to do a check and replace all for ROAMFLAG_EVENT, pretty sure we have used this elsewhere.

Reply:
I did a full repo search for ROAMFLAG_EVENT and I'm finding zero occurrences.